### PR TITLE
reddit.com - Add filter for "Download the app" popup

### PR DESCRIPTION
**Pull request by DerrickGibbs1**
Popup is blocking access to the page and cannot be dismissed

[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***: Block a new popup on reddit.
* **Current behaviour**: reddit has a new popup that is blocking access to the page on mobile browsers.

### CODE REVIEW

**Comment by zloyden** [comment]
@DerrickGibbs1 can you please attach a screenshot with this element? Unfortunately, I can't reproduce it.

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -156,6 +156,7 @@ muzofond.fm###sveta__muzika-banner
 m.kenh14.vn##a[href="https://kenh14.vn/download-app.htm"] > img
 moviesjoy.*,himovies.to,tinyzonetv.to###gift-top
 reddit.com##.xPromoAppStoreFooter
+reddit.com##.XPromoBlockingModal
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:159]: @DerrickGibbs1 can you please attach a screenshot with this element? Unfortunately, I can't reproduce it.

**Comment by DerrickGibbs1** [comment]
It looks like reddit is showing this ad only on small subs and only on mobile. For example, if you visit https://www.reddit.com/r/videogamephotography/ on an iPhone or with an iPhone user agent it will show up. You may have to reload the page a couple of times. Since posting this filter I have discovered that it does not seem to work with adgaurd for iOS, despite working in uBlock on firefox. There may be a syntax error I do not understand. As you can see this popup blocks the screen and there is no way to dismiss.

screenshot of ad

**Comment by zloyden** [comment]
Thanks for the link, I found it. This works fine on my end, so it can be merged.

**Review by zloyden — APPROVED** [review]

**Pull request merged**